src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -394,19 +394,20 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 
 		(ParamFloat<px4::params::FW_T_CLMB_MAX>) _param_fw_t_clmb_max,
 		(ParamFloat<px4::params::FW_T_HRATE_FF>) _param_fw_t_hrate_ff,
-		(ParamFloat<px4::params::FW_T_HRATE_P>) _param_fw_t_hrate_p,
-		(ParamFloat<px4::params::FW_T_INTEG_GAIN>) _param_fw_t_integ_gain,
+		(ParamFloat<px4::params::FW_T_ALT_TC>) _param_fw_t_h_error_tc,
+		(ParamFloat<px4::params::FW_T_I_GAIN_THR>) _param_fw_t_I_gain_thr,
+		(ParamFloat<px4::params::FW_T_I_GAIN_PIT>) _param_fw_t_I_gain_pit,
 		(ParamFloat<px4::params::FW_T_PTCH_DAMP>) _param_fw_t_ptch_damp,
 		(ParamFloat<px4::params::FW_T_RLL2THR>) _param_fw_t_rll2thr,
 		(ParamFloat<px4::params::FW_T_SINK_MAX>) _param_fw_t_sink_max,
 		(ParamFloat<px4::params::FW_T_SINK_MIN>) _param_fw_t_sink_min,
 		(ParamFloat<px4::params::FW_T_SPD_OMEGA>) _param_fw_t_spd_omega,
 		(ParamFloat<px4::params::FW_T_SPDWEIGHT>) _param_fw_t_spdweight,
-		(ParamFloat<px4::params::FW_T_SRATE_P>) _param_fw_t_srate_p,
+		(ParamFloat<px4::params::FW_T_TAS_TC>) _param_fw_t_tas_error_tc,
 		(ParamFloat<px4::params::FW_T_THR_DAMP>) _param_fw_t_thr_damp,
-		(ParamFloat<px4::params::FW_T_THRO_CONST>) _param_fw_t_thro_const,
-		(ParamFloat<px4::params::FW_T_TIME_CONST>) _param_fw_t_time_const,
 		(ParamFloat<px4::params::FW_T_VERT_ACC>) _param_fw_t_vert_acc,
+		(ParamFloat<px4::params::FW_T_STE_R_TC>) _param_ste_rate_time_const,
+		(ParamFloat<px4::params::FW_T_TAS_R_TC>) _param_tas_rate_time_const,
```

Comment:
Why not directly exposing the P gain (1/Tc) to the user? People are more used to tune P gains than "time constants" (even if it's the same) and it would save some unnecessary conversions.

Reply:
@bresch I'm having trouble agreeing honestly. Not only because it always used to be a time constant but also because personally I find time constants much more intuitive than P gains. You are right in the sense that one is the inverse of the other but I find that it's easier to tell a person that if he sets the time constant to 10 seconds, then he can expect altitude errors not to be corrected faster than that. So even though it's not very precise in therms of mathematics I still feel it's more intuitive.

Reply:
Anything to make it easier to understand will go a long way. _"...tell a person that if he sets the time constant to 10 seconds, then he can expect altitude errors not to be corrected faster than that. "_ Is easy to understand. Why not just also say... this is the same as a P gain then you have it both for people

Reply:
Ok, I can understand that. Let's change the I gains to integral time constants then?

Reply:
@bresch How does it make the I gain more intuitive for the user? How will you explain to him what exactly the time constant for the integrator does?

Reply:
![DeepinScreenshot_select-area_20201211103542](https://user-images.githubusercontent.com/14822839/101887093-b0147600-3b9c-11eb-9010-fc329d18ea37.png)

Reply:
@MaEtUgR Yes, mathematically that makes sense, the question is if it tells the user anything.